App/src/test/java/cc/softwarefactory/lokki/android/espresso/PlacesScreenTest.java

```diff
@@ -74,6 +84,23 @@ public void testContactAppearsInPlace() throws JSONException {
     }
 
 
+    public void testClickContactOpensMap() throws JSONException, InterruptedException {
+        getMockDispatcher().setPlacesResponse(new MockResponse().setBody(MockJsonUtils.getPlacesJson()));
+        String[] contactEmails = (new String[]{"family.member@example.com"});
+        JSONObject location = new JSONObject();
+        location.put("lat", "37.483477313364574") //Testplace1
```

Comment:
What's this comment? Do we use these coordinates elsewhere as well? We probably want to put them into constant in that case.